AzureTable/Microsoft.DataTransfer.AzureTable.Wpf/Source/AzureTableSourceAdapterConfigurationSummaryPage.xaml

```diff
@@ -25,30 +25,36 @@
       <RowDefinition />
       <RowDefinition />
       <RowDefinition />
+      <RowDefinition />
     </Grid.RowDefinitions>
 
     <Label Grid.Row="0" Grid.Column="0" Content="{DynamicResource ConnectionStringSummaryKey}" />
     <Label Grid.Row="0" Grid.Column="1">
       <TextBlock Text="{Binding ConnectionString}" />
     </Label>
 
-    <Label Grid.Row="1" Grid.Column="0" Content="{DynamicResource TableNameSummaryKey}" />
+    <Label Grid.Row="1" Grid.Column="0" Content="{DynamicResource LocationModeSummaryKey}" />
     <Label Grid.Row="1" Grid.Column="1">
-      <TextBlock Text="{Binding Table}" />
+        <TextBlock Text="{Binding LocationMode}" />
```

Comment:
As this seems to be an enum, it might make sense to use localizable, human readable text, instead of enum value name. Take a look at CollectionTierConverter on [this page](/Azure/azure-documentdb-datamigrationtool/blob/master/DocumentDb/Microsoft.DataTransfer.DocumentDb.Wpf/Sink/Bulk/DocumentDbBulkSinkAdapterConfigurationSummaryPage.xaml).